### PR TITLE
18_API getting random job

### PR DESCRIPTION
**Pull request by Jean-Beru**
Issue #13

### CODE REVIEW

File: app/config/parameters_dev.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+parameters:
+    sensiolabs_jobboard.api_allowed_hosts: ~
```
Comment (salahm) [inline app/config/parameters_dev.yml]: why this is not in `config_dev.yml` ?

File: app/config/security.yml [hunk 2]
```diff
@@ -32,3 +32,4 @@ security:
 
     access_control:
         - { path: ^/manage, roles: ROLE_CONNECT_USER }
+        - { path: ^/api, roles: API_READ }
```
Comment (salahm) [inline app/config/security.yml]: `API_` is not a role

**Comment by salahm** [comment]
for host matching, you don't have to reinvent the weel, you can use existing host requirement for routes
http://symfony.com/doc/current/components/routing/hostname_pattern.html

**Comment by Jean-Beru** [comment]
Correct. It's simpler like this!

File: app/config/config_test.yml [hunk 3]
```diff
@@ -18,3 +18,6 @@ swiftmailer:
 doctrine:
     dbal:
         dbname:   %database_name%_test
+
+parameters:
+    job_board.allowed_host: valid_host.com|other_valid_host.com
```
Comment (salahm) [inline app/config/config_test.yml]: why are changing these values here ?

**Comment by salahm** [comment]
:+1:

File: app/config/config_test.yml (hunk 3 above)
Comment (Jean-Beru) [inline app/config/config_test.yml]: I changed them in order to test host restriction.
Cf. https://github.com/poledev/hubert/blob/issue-13/src/SensioLabs/JobBoardBundle/TestsFunctional/Controller/ApiControllerTest.php#L23

**Pull request merged**